.github/actions/ensure-image/action.yml

```diff
@@ -0,0 +1,38 @@
+name: Ensure OpenStack image
+
+description: >-
+  Ensures that the specified image exists in the target OpenStack cloud.
+
+inputs:
+  os-client-config-file:
+    description: The path of the OpenStack clouds file
+    required: true
```

Comment:
should this be required if there is a default?

Reply:
See below